### PR TITLE
perf(ext/websocket): Cut event dispatch/blob overhead as much as we can

### PR DESCRIPTION
**Pull request by mmastrac**
Small improvement, seems to reduce benchmark jitter a bit. I was not able to see any major differences with `--trace-gc`, but the results are consistently better (average is higher and std dev is much lower)

Before:

```
Msg/sec: 155174.750000
Msg/sec: 160164.250000
Msg/sec: 164021.750000
Msg/sec: 164089.750000
Msg/sec: 161876.000000
Msg/sec: 164591.750000
Msg/sec: 166381.000000
Msg/sec: 165368.500000
Msg/sec: 152099.000000
Msg/sec: 160498.000000
```

After:

```
Msg/sec: 165066.250000
Msg/sec: 163997.750000
Msg/sec: 166492.500000
Msg/sec: 167396.500000
Msg/sec: 167681.500000
Msg/sec: 164960.500000
Msg/sec: 165815.000000
Msg/sec: 166636.250000
Msg/sec: 166958.000000
Msg/sec: 166451.250000
Msg/sec: 166227.750000
Msg/sec: 165989.750000
```

### CODE REVIEW

File: ext/websocket/01_websocket.js [hunk 1]
```diff
@@ -436,7 +437,7 @@ class WebSocket extends EventTarget {
             origin: this[_url],
             [_skipInternalInit]: true,
           });
-          dispatch(this, event);
+          dispatchFast(this, event);
```
Comment (bartlomieju) [inline ext/websocket/01_websocket.js:440]: There are other places in this file where we could use this new function

**Review by littledivy — APPROVED** [review]
LGTM!

**Review by bartlomieju — CHANGES_REQUESTED** [review]
We discussed this offline yesterday and we're not sure we want to do it. It actually breaks web compatibility for events emitted from WebSockets. We'll discuss it further before merging.

**Comment by bartlomieju** [comment]
I'm gonna close this PR as we discussed other solutions to this problem and we are gonna explore them first.

**Pull request closed without merging**